core/modules/editor/operations/text/wrap-selection.js

```diff
@@ -13,37 +13,121 @@ Text editor operation to wrap the selection with the specified prefix and suffix
 "use strict";
 
 exports["wrap-selection"] = function(event,operation) {
-	if(operation.selStart === operation.selEnd) {
-		// No selection; check if we're within the prefix/suffix
-		if(operation.text.substring(operation.selStart - event.paramObject.prefix.length,operation.selStart + event.paramObject.suffix.length) === event.paramObject.prefix + event.paramObject.suffix) {
+	var o = operation,
```

Comment:
I'm not a huge fan of this abbreviation pattern. I understand that it saves a bit of space, but it is hard to apply consistently. For example, here one might wonder why "operation" is abbreviated but "prefix" is not.

On balance, I would prefer to find ways to refactor the code to avoid the repetition, rather than use this pattern.

Reply:
prefix is an abbreviation. It is the shorthand for `event.paramObject.prefix`

Reply:
The "o" abbreviation is used 99 times and was introduced to make the code readable and understandable. If I add it back again, it will make the code completely unreadable. Reducing the file size by 800bytes is a side effect. 

IMO there is no redundant, repettitive code on the right side of any =. Every line, that defines o.cutStart, o.cutEnd, ... and so on, has a different pattern on the right side. -- So I do not really see a possility to optimize that any further.